avni-server-api/src/main/java/org/avni/server/domain/metabase/GroupPermissionsGraphResponse.java

```diff
@@ -0,0 +1,42 @@
+package org.avni.server.domain.metabase;
+
+import java.util.Map;
+
+import org.springframework.stereotype.Component;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@Component
```

Comment:
Why is this marked as a component

Reply:
This is needed,because  it is injected into GroupPermissionsService, which requires Spring to manage its lifecycle and dependency injection, without it, we get an error due to the missing Spring bean